src/plugins/intel_npu/src/plugin/npuw/partitioning/online/utils/utils.cpp

```diff
@@ -44,33 +44,59 @@ std::string ov::npuw::online::util::repeated_id(const std::shared_ptr<Repeated>&
     return ss.str();
 }
 
-std::optional<ov::npuw::online::Avoid> ov::npuw::online::util::parseAvoid(const std::string& s) {
+std::tuple<ov::npuw::online::PatternType, std::string, std::string> ov::npuw::online::util::parse(
+    const std::string& s) {
     auto pos_col = s.find(':');
     auto pos_sl = s.find('/');
 
     if (pos_col == std::string::npos || pos_sl == std::string::npos) {
-        LOG_WARN("Incorect pattern in OPENVINO_NPUW_AVOID: "
-                 << s << ". Please, separate a device with / and pattern type with :, e.g. Op:Select/NPU,P:RMSNorm/NPU."
-                 << " Avoid rule " << s << " is ommited!");
-        return std::nullopt;
+        LOG_WARN("Incorrect pattern in OPENVINO_NPUW_AVOID or OPENVINO_NPUW_ISOLATE: "
```

Comment:
Ideally this method shouldn't be aware where it is used, so giving this hint here may be a bad idea. Just say "Incorrect pattern {s}"

Reply:
Done